spec/shortcuts_spec.rb

```diff
@@ -159,6 +159,38 @@
     end
   end
   
+  context "::magic" do
+    it "creates a matrix with numbers from 1 to n^n(n squared)" do
```

Comment:
@shahsaurabh0605 I believe the other data types are  `:int16,:int32,:int64` .  So Adding something similar to   
`data_types=[:int8,:int16,:int32,:int64]`
    `data_types.each do |dtype|` will help you run the spec tests for all data types.